ts/features/itwallet/navigation/useItwLinkingOptions.tsx

```diff
@@ -12,6 +13,13 @@ import { ITW_ROUTES } from "./routes";
 export const useItwLinkingOptions = (): PathConfigMap<AppParamsList> => {
   const isItwValid = useIOSelector(itwLifecycleIsValidSelector);
   const isItwEnabled = useIOSelector(isItwEnabledSelector);
+  const isMdlPresent = useIOSelector(itwHasMdlCredentialSelector);
```

Comment:
Instead of making this check here in the linking options, we could replace the `ItwPresentationCredentialDetailScreen` content when the `itwCredentialByTypeSelector(credentialType)` selector returns `none`. 
I think it's a cleaner implementation. What do you think?

Reply:
Yes, I think this should work. I will remove the new screen and implement the change inside `ItwPresentationCredentialDetailScreen`. Thanks!